asserts/device_asserts.go

```diff
@@ -194,3 +194,61 @@ func assembleSerial(assert assertionBase) (Assertion, error) {
 		pubKey:        pubKey,
 	}, nil
 }
+
+// SerialRequest holds a serial-request assertion, which is a self-signed request to obtain a full device identity bound to the device public key.
+type SerialRequest struct {
+	assertionBase
+	pubKey PublicKey
+}
+
+// BrandID returns the brand identifier of the device making the request.
+func (sreq *SerialRequest) BrandID() string {
+	return sreq.HeaderString("brand-id")
+}
+
+// Model returns the model name identifier of the device making the request.
+func (sreq *SerialRequest) Model() string {
+	return sreq.HeaderString("model")
+}
+
+// NonceTicket returns the nonce/ticket for the request.
+func (sreq *SerialRequest) NonceTicket() string {
+	return sreq.HeaderString("nonce-ticket")
```

Comment:
The story there is that is actually both indeed, it's both a nonce (at the first moment you present this), and then it also plays a role of a ticket (but not a nonce anymore) if the brand service couldn't answer immediately with a serial (202 vs 200)